crates/fetch_queue/src/fetch_task.rs

```diff
@@ -0,0 +1,52 @@
+use std::time::Duration;
+
+use kitsune2_api::fetch::{DynFetchQueue, FetchTaskT};
+
+/// Configuration for [`FetchTask`].
+pub struct FetchTaskConfig {
```

Comment:
The [peer-store pr](https://github.com/holochain/kitsune2/pull/19/files#diff-72a2ccbabb7487b60b153d720970a3bf4ce28548158ec52e4ac35a4a93b00e8dR9-R16) has an example of a module config and how to work with it in the factory.

Reply:
When I wrote this, the peer store PR didn't exist yet and I was hesitant to introduce the factory. I see now how it's done.